docs/data-process.rst

```diff
@@ -253,6 +253,46 @@ and the output will look like:
        - martian
 
 
+Multiple Input Files Concatenation
+----------------------------------
+
+You can specify multiple input files in both command line and Python usages.
+The input files will be concatenated together (as text) before before parsed as
+a whole YAML document. For example, suppose we have ``part1.yaml`` with:
+
+.. code-block:: yaml
+
+   hello:
+
+And ``part2.yaml`` with:
+
+   - earth
+   - mars
+
+And ``part3.yaml`` with:
+
+   - jupiter
+   - saturn
+
+Running :program:`yp-data -o- part1.yaml part2.yaml part3.yaml` will give:
```

Comment:
I don't think -o- is correct?

Reply:
and also don't you need to specify an output file?

Reply:
`-o-` sends the output to STDOUT.

Reply:
Oh I see my misunderstanding.  Perhaps wroth a comment which says this.  I'll approve.